src/test/java/org/folio/innreach/controller/ContributionCriteriaControllerTest.java

```diff
@@ -0,0 +1,93 @@
+package org.folio.innreach.controller;
+
+import org.folio.innreach.controller.base.BaseControllerTest;
+import org.folio.innreach.dto.ContributionCriteriaDTO;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+import java.util.Set;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.folio.innreach.fixture.TestUtil.deserializeFromJsonFile;
+
+class ContributionCriteriaControllerTest extends BaseControllerTest {
+  private static Set<String> SKIP_BEFORE_EACH_FOR_TESTS = Set.of(
+    "return200HttpCode_on_createCriteriaConfiguration()"
+    , "return404HttpCode_when_deletableCriteriaConfiguration_DoesNotExist()");
+
+  private static final String PRE_POPULATED_CENTRAL_SERVER_ID = "efb089d4-4416-4892-ab81-bdfa00e4a2c3";
+
+  @Autowired
+  private TestRestTemplate testRestTemplate;
+
+  @BeforeEach
+  void setUp(TestInfo testInfo) {
+    if (SKIP_BEFORE_EACH_FOR_TESTS.contains(testInfo.getDisplayName())) return;
+    var contributionCriteriaDTO = deserializeFromJsonFile("/contribution-criteria/create-contribution-configuration-request.json", ContributionCriteriaDTO.class);
+    var responseEntity = testRestTemplate.postForEntity("/inn-reach/central-servers/contribution-criteria/", contributionCriteriaDTO, ContributionCriteriaDTO.class);
+    assertEquals(HttpStatus.CREATED, responseEntity.getStatusCode());
+  }
+
+  @AfterEach
+  void tearDown() {
+    var responseEntity = testRestTemplate.exchange("/inn-reach/central-servers/{centralServerId}/contribution-criteria",
```

Comment:
The same here, please, consider the @Sql annotation

Reply:
Andrew, please explain why I must use native SQL scripts instead Hibernate call?